UI/app/src/main/java/com/example/pathfinder/Activities/DashboardActivity.java

```diff
@@ -9,12 +9,9 @@
 import android.view.View;
 import android.view.WindowManager;
 import android.widget.ImageView;
-import android.widget.RelativeLayout;
 import android.widget.TextView;
 import android.widget.Toast;
-import android.widget.ProgressBar;
 import android.widget.SeekBar;
-import android.widget.ToggleButton;
```

Comment:
I'm not sure what the ToggleButton was for, could it be part of the SeekBar? In this case it might need to be put back

Reply:
I actually don't know haha, the button was not used anywhere so I deleted it. 
🙊